device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/config.bcm.j2

```diff
@@ -1,16 +1,23 @@
 {# Construct config.bcm to include additional soc properties per specific device metadata requirement #}
 {%- set map_prio = '' -%}
+{%- set pfcwd_sock = '' -%}
 {%- if DEVICE_METADATA is defined and DEVICE_METADATA['localhost'] is defined and DEVICE_METADATA['localhost']['subtype'] is defined -%}
 {%-     set switch_subtype = DEVICE_METADATA['localhost']['subtype'] -%}
 {%-     if 'dualtor' in switch_subtype.lower() %}
 {%-         set map_prio = 'sai_remap_prio_on_tnl_egress=1' -%}
 {%-     endif %}
 {%- endif %}
+{%- if SYSTEM_DEFAULTS is defined and 'tunnel_qos_remap' in SYSTEM_DEFAULTS and SYSTEM_DEFAULTS['tunnel_qos_remap']['status'] == 'enabled' -%}
+{%-     set pfcwd_sock = 'hybrid_pfc_deadlock_enable=1
+                          pfc_deadlock_seq_control=1
+                          sai_pfc_dlr_init_capability=1' -%}
```

Comment:
`
{%-     set pfcwd_sock = 
'hybrid_pfc_deadlock_enable=1
pfc_deadlock_seq_control=1
sai_pfc_dlr_init_capability=1' -%}
`

Reply:
addressed in #15240